src/dynsight/_internal/track/track.py

```diff
@@ -46,20 +48,24 @@ def track_xyz(
             Path where the output .xyz file with particle IDs will be saved.
 
         search_range:
-            Maximum linking distance between frames.
+            The maximum allowable displacement of objects between frames for
+            them to be considered the same particle.
 
         memory:
-            Maximum number of frames a particle can vanish and still be
-            re-identified.
-
-        adaptive_step:
-            Factor by which the search range is multiplied to reduce it during
-            adaptive search.
+            The maximum number of frames during which an object can vanish,
+            then reppear nearby, and be considered the same particle.
 
         adaptive_stop:
-            Minimum allowable search range during adaptive search. If the
-            search range becomes smaller than this value and ambiguities
-            persist, the linking process is aborted for the problematic region.
+            If not `None`, when encountering a region with too many candidate
+            links (subnet), retry by progressively reducing `search_range`
+            until the subnet is solvable. If `search_range` becomes less or
+            equal than the `adaptive_stop`, give up and raise a
+            `SubnetOversizeException`.
+
+        adaptive_step:
+            Factor by which the `search_range` is multiplied to reduce it
+            during adaptive search. Effective only if `adaptive_stop` is not
+            `None`.
```

Comment:
I think this should also be None by default then, and an error  should raise if not assigned when adaptive_stop is not None, or if assigned and adaptive_stop is None